velox/common/tests/BitUtilTest.cpp

```diff
@@ -416,6 +416,43 @@ TEST_F(BitUtilTest, forEachBit) {
   expectedIndex = 3;
   forEachUnsetBit(data, 2, totalBits - 3, calledOnEveryOther);
   EXPECT_EQ(totalBits - 3, expectedIndex);
+
+  auto numEntries = 100;
```

Comment:
Entries is a strange term to use here. Let's use bits. Perhaps, just reuse totalBits variable.